### DISCUSSION TITLE
Gamification addon

### DISCUSSION THREAD

**Discussion by 0a1b** [iSoron/uhabits#2031] [Feature Requests]
I think gamification can help a great deal to reach ones goals. I am thinking of sth as simple as badges one can get for streakes, like 10,25,50,75... 1000, etc days and badges total day for each habit and maybe some for generally sticking to ones habits.
The badges don't have to be too fancy but I guess this would boost adherence to the habit and the app.

**Comment by KyleSCraig** [comment]
Personally I have had to struggle with removing myself from the existing gamification. Please don't gamify this app, especially with badges.

**Comment by 0a1b** [comment]
My suggestion was an add-on, hence something that can be activated and
deactivated.

KyleSCraig ***@***.***> schrieb am Mi., 14. Aug. 2024, 01:06:
